gbmodel/model_sqlalchemy.py

```diff
@@ -101,8 +101,11 @@ def get_team_session_id(self, session_id):
         Input: session id of the selected session
         Output: list of teams and their info. from the selected session
         """
-        team = teams.query.filter_by(session_id=session_id).all()
-        return team
+        try:
+            team = teams.query.filter_by(session_id=session_id).all()
+            return team
+        except exc.SQLAlchemyError:
+            return None
```

Comment:
I decided to add some try/except while I was here. I didn't think it would hurt.